docs/data-sources/confluent_schema_registry_cluster.md

```diff
@@ -75,11 +75,10 @@ In addition to the preceding arguments, the following attributes are exported:
 - `kind` - (Required String) A kind of the Schema Registry cluster, for example, `Cluster`.
 - `rest_endpoint` - (Required String) The HTTP endpoint of the Schema Registry cluster, for example, `https://psrc-00000.us-west-2.aws.confluent.cloud`.
 - `private_rest_endpoint` - (Required String) The private HTTP endpoint of the Schema Registry cluster, for example, `https://lsrc.us-west-2.aws.private.confluent.cloud`.
+- `private_regional_rest_endpoint` - (Required Map) The private regional HTTP endpoint map of the Schema Registry cluster.
```

Comment:
```suggestion
- `private_regional_rest_endpoints` - (Required Map) The private regional HTTP endpoint map of the Schema Registry cluster, for example:
```
{
    "us-central-1": "https://lsrc-abc.us-central-1.aws.private.confluent.cloud",
    "us-west-2": "https://lsrc-abc.us-west-2.aws.private.confluent.cloud"
}
```
```

Reply:
The reasoning is
```
private_regional_rest_endpoints['us-west-2']
```
looks a bit better than
```
private_regional_rest_endpoint['us-west-2']
```
Especially given that the private_rest_endpoint attribute is a string, whereas private_regional_rest_endpoints is a dictionary.